src/index.html

```diff
@@ -11,8 +11,63 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
```

Comment:
There are duplicate `preconnect` links to Google fonts. You only need one set of `preconnect` links for `https://fonts.googleapis.com` and `https://fonts.gstatic.com`.